docs/development/codeguide.rst

```diff
@@ -728,11 +525,11 @@ In this case, ``packagename/__init__.py`` may be::
 
 This allows functions or classes in the submodules to be used directly as
 ``packagename.foo`` rather than ``packagename.submodule1.foo``. If this is
-used, it is strongly recommended that the submodules make use of the __all__
-variable to specify which modules should be imported. Thus, submodule2.py
+used, it is strongly recommended that the submodules make use of the ``__all__``
+variable to specify which modules should be imported. Thus, ``submodule2.py``
 might read::
 
-    from numpy import array,linspace
+    from numpy import array, linspace
 
     __all__ = ('foo', 'AClass')
```

Comment:
Can we make this a list, not a tuple? This seems standard practice in astropy...